create-reproduction.md

```diff
@@ -1,29 +1,71 @@
+By creating a reproduction repository, a maintainer should have _all_ of the same information.
 
-Maybe you even figure out it is not an actual bug at all, but an unrelated problem that only manifests in your project because of other reason.
+The goal of the reproduction repository is to replicate the bug in a minimal project that contains only the necessary platforms, libraries, and plugins to reproduce the bug. Often, being able to replicate a bug outside of the usual project will lead to explaining more information.
 
-## How?
+When creating the minimal project, it may be possible to discover that it is not a bug, but something unrelated and only manifests in the main project for other reasons.
 
-1. Execute the command `cordova create` to create a new project locally
-1. Create a public Git repository on e.g. GitHub or Bitbucket and initialize a repository with that remote in your newly created Cordova project
-1. Commit that clean and empty project to your Git repository
-1. Then add the platforms and/or plugins needed to reproduce the problem
-1. Commit the changes in the project again and include the exact commands you executed in the commit message
-1. Then change the code of the sample project so it triggers your problem
-1. Finally commit the reproduction code for your bug to Git and post the repository to your issue
+## How
 
-The Cordova maintainer then can check out your Git repository, and just run the commands you described to actually see and experience the bug.
+The steps below show how to create a reproduction repository, on GitHub. This repository is for Apache Cordova maintainers for reproducing, evaluating, and debugging the associated issue ticket. Any public accessible git remote service is acceptable. For example GitHub, BitBucket, GitLab, etc.
 
-The repository later can also be a "test" to confirm the problem is actually resolved in an updated version of whatever library is faulty.
+### Step 1: Creating a Blank GitHub Repository
+1. Go to (Create a new repository)[https://github.com/new] on GitHub.
+1. Insert a `Repository name`
+1. Select `Public` to make the repository public accessible
+1. Click `Create repository`
+
+**IMPORTANT:** Do not check `Initialize this repository with a README`
+
+### Step 2: Uploading the Clean Project
+In the terminal, execute the following commands:
+
+```
+$ cordova create reproduction-sample
+$ cd reproduction-sample
+$ git init
+$ git add .
+$ git commit -m "Initialize with Clean Project"
+$ git remote add origin git@github.com:<ACCOUNT>/<REPO_NAME>.git
+$ git push -u origin master
+```
+
+### Step 3: Adding Necessary Dependencies
+Add and commit on each necessary platform(s) and/or plugin(s) needed to reproduce the problem.
```

Comment:
I don't think that committing each time is necessary. :man_shrugging:

Reply:
I also think it would be OK with just one commit and a commit message containing the order of commands. I feel this could be tedious for the user but I heard differently... 🙈

Reply:
Yeah @janpio be easy on the users that actually made it this far :stuck_out_tongue:

Reply:
Committing once before doing the actual code changes is nice, but especially with 17 plugins this can be helpful as well. But I'm ok either way - as long as the code changes and the plugin/platform stuff are both separate from the initial commit.

Reply:
Whatever works for you better. When you are ready, just merge it into my branch and we will continue in the original PR.

Reply:
@raphinesse @janpio Here is an option, will this fit for everyone? Since writing per commit provides granularity but can be tedious, what if we add an alternative statement.

```Alternatively, it is possible to bundle the plugin(s) and platform(s) into a single commit. The commit message must include the commands that had been executed in the exact order. One of the drawbacks of bundling into a single commit is not being able to see all changes at a granular level. For example, some plugins have hook scripts that modify the platform build code which can lead to issues.```

This provides two acceptable options and also explains why the 1 commit per action can be better.

Reply:
I'd rather keep it simple than give options. But I'm fine either way.

One thing that text made me think of: We had recently been trying to fix `cordova create` to also generate a `.gitignore`. It seems in this case it would be better if no files were ignored so you see all the stuff happening in `platforms`, as mentioned by @erisu. So we might want to add a step that says "delete `.gitignore` if present".

Reply:
I'm fine with or without that added text. 
Actually, I probably prefer without to keep the document short(er).
People will do whatever they want anyway - which is fine in this case.